packages/js/product-editor/src/components/iframe-editor/utils/are-blocks-empty.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { BlockInstance } from '@wordpress/blocks';
+
+/**
+ * By default the blocks returned by the editor contains one paragraph
+ * block with empty content. This function checks to see if the blocks
+ * consists of a single paragraph block with empty content.
+ *
+ * @param blocks The block list
+ * @return true if the blocks consists of a single paragraph block with empty content; false otherwise
```

Comment:
This also returns true if there are no blocks, right? So the documentation is a bit confusing.